packages/yourloops/pages/hcp/team-card.tsx

```diff
@@ -82,6 +82,8 @@ function TeamCard(props: TeamCardProps): JSX.Element {
   const [buttonsDisabled, setButtonsDisabled] = React.useState(false);
 
   const { id } = team;
+  const isTeamMemberAdmin = memberRole === TeamMemberRole.admin && memberStatus === UserInvitationStatus.accepted;
```

Comment:
Maybe changing this file was a little out of scope ?
There is enough changed already I think, it can be done in another PR.